### DISCUSSION TITLE
How to migrate from pyhomematic to hahomematic?

### DISCUSSION THREAD

**Discussion by msinn** [SukramJ/aiohomematic#1281] [General]
Hi,
I am using pyhomematic for interfacing our smarthome automation project [SmartHomeNG](https://github.com/smarthomeng) to HomeMatic. Since pyhomematic is being retired and hahomematic is intended to become the successor of [pyhomematic](https://github.com/danielperna84/pyhomematic), I want to know if there is any documentation how to use hahomematic outside the context of Home Assistant or if there is any documentation how to migrate.

**Comment by SukramJ** [comment]
There is no documentation, and there is no plan to create this documentation, at least not from me.
If you need an entry point, take a look at example.py or the custom integration [Homematic(IP) Local](https://github.com/danielperna84/custom_homematic).
Even if `pyhomematic` and `hahomematic` are similar in their very rough concept, `hahomematic` is a completely new development, and the only goal is the best possible support for the custom integration (Homematic(IP) Local) into Homeassistant.
Unfortunately, I can't tell you how to migrate from pyhomematic to hahomematic because I don't know pyhomematic well and haven't thought about such a migration.
